### PR TITLE
PSPs: add annotation to allow seccomp profiles

### PR DESCRIPTION
**Pull request by hervenicol**
Towards https://github.com/giantswarm/giantswarm/issues/27522

### CODE REVIEW

**Comment by hervenicol** [comment]
Tested on a WC on Gremlin :+1:

File: helm/prometheus-operator-app/values.yaml [hunk 1]
```diff
@@ -8,6 +8,10 @@ ciliumNetworkPolicy:
   enabled: false
 
 prometheus-operator-app:
+  global:
+    rbac:
+      pspAnnotations:
+        seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
```
Comment (QuentinBisson) [inline helm/prometheus-operator-app/values.yaml:14]: Can you check that this works also for ksm, node-exporter and grafana?

Comment (hervenicol) [reply helm/prometheus-operator-app/values.yaml:14]: > Can you check that this works also for ksm, node-exporter and grafana?

...and alertmanager.
Yep, works :+1:

Comment (QuentinBisson) [reply helm/prometheus-operator-app/values.yaml:14]: Nice :)

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**